pipeline/types/notification.py

```diff
@@ -8,6 +9,10 @@
 class NotificationInput:
     # relevant chat data between the users in the given match
     chatdata: ChatData
+    # datetime to inform Notifier class when pipeline is being run
+    run_datetime: datetime.datetime
+    # the type of notification
+    notification_type: str = ""
```

Comment:
Let's make `notification_type` an enum instead, that way we can define explicit types and see all the options. Search the codebase for `Enum` to see some examples.